src/main/scala/Game.scala

```diff
@@ -0,0 +1,94 @@
+class Game {
+  var player1: Player = _
+  var player2: Player = _
+
+  def startGame(): Unit = {
+    menu()
+
+    val roundsStr = scala.io.StdIn.readLine("How many rounds? ")
+    val rounds = roundsStr.toInt
+
+    do {
+      // Store the players moves - Ai will make decisions based on opposition moves
+      player1.getOppositionMoves = player2.moveList
+      player1.genMove()
+
+      player2.getOppositionMoves = player1.moveList
+      player2.genMove()
+
+      println(player1.name + ": " + player1.currentMove)
+      println(player2.name + ": " + player2.currentMove)
+      checkWinner(player1, player2)
+      displayScores()
+      println()
+
+    } while (player1.score != rounds && player2.score != rounds)
+
+    theEnd()
+
+  }
+
+  def menu(): Unit = {
```

Comment:
this could be split into two methods, one that will be printing all of this out, another that will determine the game mode